dataplane/type.h

```diff
@@ -208,6 +208,11 @@ struct ipv6_address_t
 		memset(bytes, 0, std::size(bytes));
 	}
 
+	bool operator<(ipv6_address_t other) const
+	{
+		return std::memcmp(bytes, other.bytes, 16) < 0;
+	}
+
```

Comment:
why is that operator needed?

Reply:
ipv6_address_t is used as real for chash. For chash to resolve collisions consistently reals need to have some ordering. < seemed to be trivial choice.